unit_test/tstBatchedLinearAlgebra.hpp

```diff
@@ -865,7 +867,8 @@ TEST( TEST_CATEGORY, kernelTest )
     kernelTest<20>();
 }
 
-TEST( TEST_CATEGORY, eigendecomposition_test ) { eigendecompositionTest(); }
+// FIXME_KOKKOSKERNELS
+// TEST( TEST_CATEGORY, eigendecomposition_test ) { eigendecompositionTest(); }
```

Comment:
@terwin FYI. You will need to uncomment this for your test. We are technically patching KokkosKernels to get the eigendecomposition stuff so we aren't including here because the CI doesn't apply the patch.

Reply:
You can also disable tests by prepending the test name with `DISABLED_`. Then the test is still built, just not run by default.

Reply:
I didn't know that. Although this one fails at compile time